jerry-core/api/jerry.c

```diff
@@ -3455,6 +3455,37 @@ jerry_get_promise_state (const jerry_value_t promise) /**< promise object to get
 #endif /* ENABLED (JERRY_BUILTIN_PROMISE) */
 } /* jerry_get_promise_state */
 
+/**
+ * Get the well-knwon symbol represented by the given `symbol` enum value.
+ *
+ * Note:
+ *      returned value must be freed with jerry_release_value, when it is no longer needed.
+ *
+ * @return undefined value - if invalid well-known symbol was requested
+ *         well-known symbol value - otherwise
+ */
+jerry_value_t
+jerry_get_well_known_symbol (jerry_well_known_symbol_t symbol) /**< jerry_well_known_symbol_t enum value */
+{
+  jerry_assert_api_available ();
+
+#if ENABLED (JERRY_ESNEXT)
+  if (symbol >= JERRY_SYMBOL__NUMBER)
```

Comment:
Based on this you should also check if it is less then the first argument. Maybe we could directly use the `LIT_IS_GLOBAL_SYMBOL` macro here?

Reply:
we cannot use `LIT_IS_GLOBAL_SYMBOL` directly since there is an offset between the first internal magic string and the enum value.

Reply:
In the assert below it is used after calculating with the offset. Can do that here also?

Reply:
Sure!